introduction.md

```diff
@@ -9,3 +9,7 @@ thx.
 *ios
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?